homeassistant/components/climate/nest.py

```diff
@@ -38,6 +38,7 @@ def __init__(self, structure, device, temp_unit):
         self._unit = temp_unit
         self.structure = structure
         self.device = device
+        self._fan_list = ["On", "Auto"]
```

Comment:
I looks like the [Vera thermostat is doing it a bit different](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/climate/vera.py#L21). Should I mimic that?